README.md

```diff
@@ -379,7 +379,14 @@ By default, all uncaught exceptions are reported at the "error" level, except fo
 - ```AbstractController::ActionNotFound```
 - ```ActionController::RoutingError```
 
-If you'd like to customize this list, see the example code in ```config/initializers/rollbar.rb```. Supported levels: "critical", "error", "warning", "info", "debug", "ignore". Set to "ignore" to cause the exception not to be reported at all.
+If you'd like to customize this list, modify the example code in ```config/initializers/rollbar.rb```. Supported levels: "critical", "error", "warning", "info", "debug", "ignore". Set to "ignore" to cause the exception not to be reported at all. For example, to ignore 404s, and treat `NoMethodError`s as critical bugs you can use the following code:
```

Comment:
Ahh, it's an oxford comma though. So both usages are technically correct. I'll drop the comma for you anyway :)